src/applications/appeals/996/tests/utils/submit.unit.spec.js

```diff
@@ -369,6 +369,6 @@ describe('getPhone', () => {
 describe('getTimeZone', () => {
   it('should return a string', () => {
     // result will be a location string, not stubbing for this test
-    expect(getTimeZone().length).to.be.greaterThan(1);
+    expect(getTimeZone().length).to.be.greaterThan(0);
```

Comment:
Was this test failing?

Reply:
I was just testing linters here, forgot to revert this.